### DISCUSSION TITLE
Is this right? (Middlewares)

### DISCUSSION THREAD

**Discussion by mochetts** [vercel/next.js#58020] [App Router]
Am I the only one doing this? Or is there something I'm missing here?

<img width="945" alt="Screenshot 2023-11-04 at 10 10 10 AM" src="https://github.com/vercel/next.js/assets/3678598/594122cb-d1fc-44d9-bcbe-285ef2c5a566">

<img width="930" alt="Screenshot 2023-11-04 at 10 15 05 AM" src="https://github.com/vercel/next.js/assets/3678598/5f2a6912-2bd6-4578-a74c-2816260be1e9">

**Comment by matheusmatos** [comment]
Sorry, where's this PR? I can't find it on this repository :(

**Reply by mochetts** [reply]
It's an internal project I took a screenshot from.
